modules/ols-about-product-coverage.adoc

```diff
@@ -5,7 +5,12 @@
 [id="ols-about-product-coverage_{context}"]
 = About product coverage
 
-{ols-official} generates answers to questions based on the content in the official {ocp-product-title} product documentation. The documentation for the following products is not part of the {ocp-short-name} product documentation; therefore, {ols-short} has limited context for generating answers about these products:
+{ols-official} generates answers to questions based on the content in the official {ocp-product-title} product documentation. 
+
+[id="product-exceptions_{context}"]
+== Product exceptions
+
+The official {ocp-product-title} product documentation does not contain information about all {red-hat} products. Therefore, {ols-official} has limited context for generating answers about the following products:
```

Comment:
Same question about "official."

Also, maybe change the second half:

Therefore, {ols-official} will only be able to provide knowledge based on the model you choose to use with the system with respect to the following products or components: